### PR TITLE
Browser wallet 0.9.9 release

### PR DESCRIPTION
**Pull request by dg-concordium**
## Purpose

Inform users about changes in 0.9.9 release.

## Changes

Release notes

## Checklist

- [X] My code follows the style of this project.
- [X] The code compiles without warnings.
- [X] I have performed a self-review of the changes.
- [ ] I have documented my code, in particular the intent of the
      hard-to-understand areas.
- [ ] (If necessary) I have updated the CHANGELOG.

## CLA acceptance

_Remove if not applicable.

By submitting the contribution I accept the terms and conditions of the
Contributor License Agreement v1.0
- link: https://developers.concordium.com/CLAs/Contributor-License-Agreement-v1.0.pdf

- [X] I accept the above linked CLA.

### CODE REVIEW

File: source/mainnet/net/resources/release-notes-mainnet.rst [hunk 1]
```diff
@@ -11,6 +11,15 @@ Release notes - Mainnet
 
    To subscribe to updates on the Mainnet status page click **Subscribe** to get all updates or click **Get updates** to choose to get all updates or only updates for specific products.
 
+March 23, 2023
```
Comment (abizjak) [inline source/mainnet/net/resources/release-notes-mainnet.rst:14]:
```suggestion
March 28, 2023
```

File: source/mainnet/net/resources/release-notes.rst [hunk 2]
```diff
@@ -11,6 +11,15 @@ Release Notes - Testnet
 
    To subscribe to updates on the Testnet status page click **Subscribe** to get all updates or click **Get updates** to choose to get all updates or only updates for specific products.
 
+March 23, 2023
```
Comment (abizjak) [inline source/mainnet/net/resources/release-notes.rst:14]:
```suggestion
March 28, 2023
```

**Review by abizjak — APPROVED** [review]
The date is likely to be March 28.

**Review by concordium-cl — APPROVED** [review]

File: source/mainnet/net/resources/release-notes-mainnet.rst [hunk 3]
```diff
@@ -11,6 +11,15 @@ Release notes - Mainnet
 
    To subscribe to updates on the Mainnet status page click **Subscribe** to get all updates or click **Get updates** to choose to get all updates or only updates for specific products.
 
+March 28, 2023
+
+|bw| 0.9.9
```
Comment (abizjak) [inline source/mainnet/net/resources/release-notes-mainnet.rst:16]: Based on the discussion today at the daily the version will also be bumped to 0.9.10

**Pull request merged**